modules/gapi/test/gapi_frame_tests.cpp

```diff
@@ -69,6 +72,10 @@ class TestMediaNV12 final: public cv::MediaFrame::IAdapter {
         };
         return cv::MediaFrame::View(std::move(pp), std::move(ss));
     }
+    cv::util::any blobParams() const override {
+        GAPI_Assert(false && "Not implemented");
+        return {};
+    }
```

Comment:
Please drop these changes after you introduce a default implementation

Reply:
Done